src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -3331,6 +3331,43 @@ def test_aks_nodepool_add_with_ossku_ubuntu2204(self, resource_group, resource_g
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus2euap')
+    def test_aks_nodepool_add_with_ossku_ubuntu2404(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=138339236&view=results) to validate the change, once the test passed, please find the corresponding recording file from pipeline artifacts and commit it to your branch.